src/NLog.Targets.Redis/RedisTarget.cs

```diff
@@ -96,7 +97,7 @@ protected override void Write(LogEventInfo logEvent)
                     redisDatabase.Publish(key, message);
                     break;
                 default:
-                    throw new Exception("no data type defined for redis");
+                    throw new Exception("No DataType defined for RedisTarget");
```

Comment:
Since this is being changed, maybe specify the possible types?

"The required Datatype property was not defined or is invalid. Consider specifying either {nameof(RedisDataType.List)} or {nameof(RedisDataType.Channel)}."

Reply:
that's indeed better 👍